mediaqueries-4/Overview.bs

```diff
@@ -1695,6 +1695,160 @@ Color Display Quality: the 'color-gamut' feature</h3>
 	you can use this feature in a negated boolean-context fashion:
 	''not (color-gamut)''.
 
+<h3 id="dynamic-range">
+Dynamic Range: the 'dynamic-range' feature</h3>
+
+	<pre class='descdef mq'>
+	Name: dynamic-range
+	Value: standard | high
+	For: @media
+	Type: discrete
+	</pre>
+</h3>
+
+'@media/dynamic-range' represents the combination of brightness and color
```

Comment:
I would change to:

'@media/dynamic-range' represents the combination of max brightness, color depth, and contrast ratio that are supported by the UA and output device.

This makes it more consistent with the detailed definition and also more consistent with the header for color-gamut.